app/controllers/reviews_controller.rb

```diff
@@ -14,7 +13,14 @@ def new
   end
 
   def create
-    render 'reviews/create'
+    mark_task_as_completed
```

Comment:
Doesn't this produce a potential race condition - i.e. should `mark_task_as_completed` be moved into the happy path of `if submission.save` ?

Reply:
Ah, that's true, good spot! Fixed.